app/grandchallenge/profiles/views.py

```diff
@@ -63,6 +64,10 @@ def get_context_data(self, **kwargs):
         profile_user = context["object"].user
         profile_groups = profile_user.groups.all()
 
+        organizations = Organization.objects.filter(
+            members_group__in=profile_groups
```

Comment:
You could also add the admins group here.

Reply:
do you mean the editors group? admin group is not an option here, it tells me.

Reply:
Yes, sorry, inconsistent naming